src/screens/PutawayItemDetail/index.tsx

```diff
@@ -216,26 +217,36 @@ const PutawayItemDetail = () => {
                     "product.id": state.putAwayItem?.["product.id"],
                     "inventoryItem.id": state.putAwayItem?.["inventoryItem.id"],
                     "putawayFacility.id": state.putAwayItem?.["putawayFacility.id"],
-                    "putawayLocation.id": state.binToData?.id,
+                    "putawayLocation.id":state.putAwayItem?.["putawayLocation.id"],
                     "quantity": state.putAwayItem?.quantity
                 }
             ],
             "orderedBy.id": "",
-            "sortBy": null
+            "sortBy": ""
         }
         const actionCallback = (data: any) => {
             console.debug("data after submit")
             console.debug(data)
-            if (data?.length == 0) {
-                // setState({...state,
-                //     pickListItem: data,
-                //     error: 'No Picklist found',
-                // });
+            if (data?.error) {
+                showPopup({
+                    title: data.error.message ? 'Failed to submit' : null,
+                    message: data.error.message ?? 'Failed to submit',
+                    positiveButton: {
+                        text: 'Retry',
+                        callback:()=>{
+                            dispatch(submitPutawayItem(state.putAwayItem?.id as string, requestBody, actionCallback));
+                        }
+                    },
+                    negativeButtonText: 'Cancel',
+                });
             } else {
-                // setState({...state,
-                //     pickListItem: data,
-                //     error: null,
-                // });
+                showPopup({
+                    title: ' submit' ,
+                    message:'successfully submit',
+                    positiveButton: {
+                        text: 'ok',
```

Comment:
`ok` is the default text on the `positiveButton` property, I think that you could leave it as `positiveButton: {}`

Reply:
I actually don't mind being explicit here. But I think the default is Ok, not ok.

Reply:
We're going to go through all of the UIs and fix a lot of the message here, so I'm just going to remove the console.log and merge this as-is.